cli/marathon.py

```diff
@@ -96,8 +96,11 @@ def getGroupDetails(self, data):
                         if 'TCP_PORTS' in app['env']:
                             tcp_ports_value = app['env']['TCP_PORTS']
                             tcp_port_list = json.loads(tcp_ports_value).keys()
+ 
+                        if 'ENABLE_AFFINITY' in app['env']:
```

Comment:
I confirmed that the check needs to also ensure that the value of the env var is not a blank string `""`.